packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/dataset/Dataset.java

```diff
@@ -73,7 +74,7 @@ public class Dataset extends TerariumAsset {
 	 * (Optional) Unformatted metadata about the dataset
 	 **/
 	@TSOptional
-	private Object metadata;
+	private Map<String, Object> metadata;
```

Comment:
Why do we need to do that?

Reply:
The `profile_dataset` adds the card under the `"data_card"` field. So we at least need the metadata to be some "dictionary like" object.